update linux portable download link to 10.10.7

linkandzelda91 opened the pull request:
need to test still.

linkandzelda91 commented:
Okay, tested by following on a fresh deb 12, seems to work fine.

BotBlake commented:
Hello Hello GARRY - yes, this time I know its you :moyai: :laughing: 
Good catch!

Is there a reason you also removed the symlink to the `jellyfin` folder?
I honestly have never installed Jellyfin manually, so I have no Idea if Jellyfin does updating out of the box now.

Thanks again and kind regards.

linkandzelda91 commented:
> Hello Hello GARRY - yes, this time I know its you 🗿 😆 Good catch!
> 
> Is there a reason you also removed the symlink to the `jellyfin` folder? I honestly have never installed Jellyfin manually, so I have no Idea if Jellyfin does updating out of the box now.
> 
> Thanks again and kind regards.

hihihi

I am also wondering if I missed something in removing it, but when following the directions on a fresh deb 12 install extracting the .tar.gz results in a jellyfin directory directly, not a jellyfin_versionetc directory. I think maybe in previous releases it was formatted differently?

linkandzelda91 commented:
> > Hello Hello GARRY - yes, this time I know its you 🗿 😆 Good catch!
> > Is there a reason you also removed the symlink to the `jellyfin` folder? I honestly have never installed Jellyfin manually, so I have no Idea if Jellyfin does updating out of the box now.
> > Thanks again and kind regards.
> 
> hihihi
> 
> I am also wondering if I missed something in removing it, but when following the directions on a fresh deb 12 install extracting the .tar.gz results in a jellyfin directory directly, not a jellyfin_versionetc directory. I think maybe in previous releases it was formatted differently?

example of what I mean, here I cleared my working dir, downloaded and extracted with the given commands. the result is a jellyfin dir and the original .tar.gz file

```
   56  sudo rm -rf *
   57  sudo wget https://repo.jellyfin.org/files/server/linux/latest-stable/amd64/jellyfin_10.10.7-amd64.tar.gz
   58  sudo tar xvzf jellyfin_10.10.7-amd64.tar.gz
   59  history
gdewitt@jftest:/opt/jellyfin$ ls
jellyfin  jellyfin_10.10.7-amd64.tar.gz
gdewitt@jftest:/opt/jellyfin$ 
```

BotBlake reviewed (approved):
I mean, I guess 😆
